apps/dev-portal-web/docs/api/cases/invoices/amount-distribution.ru.mdx

```diff
@@ -0,0 +1,95 @@
+## Общая информация
+Средства, поступившие в результате оплаты счета, обычно поступают на банковские реквизиты, указанные при настройке Маркетплейса (если не было сделано иных корректировок).
+В случае, если нужно разделить полученные средства на разные счета (разные подрядчики, поставщики услуг и т.п.), используется поле `amountDistribution` при создании счета.
+Поле `amountDistribution` - это json массив, состоящий из объектов следующего вида:
+```json lines
```

Comment:
strange mix of json and types....

Use typescript snippet instead:
```typescript
type TypeName = {
}
```

Reply:
Fixed